grpc-sys/grpc_wrap.c

```diff
@@ -748,6 +748,9 @@ grpcwrap_server_request_call(grpc_server *server, grpc_completion_queue *cq,
                                   &(ctx->request_metadata), cq, cq, tag);
 }
 
+
+#ifdef GRPC_SYS_SECURE
```

Comment:
using GRPCIO_SYS_USE_SECURE may be better, keep the same style with other macros.

Reply:
You mean `GRPCIO_SYS_USE_PKG_CONFIG`?  It is an environment variable not a macro.  So I do not think they should keep the same style.